sound/soc/sof/topology.c

```diff
@@ -773,22 +774,27 @@ static void sof_parse_uuid_tokens(struct snd_soc_component *scomp,
 			/* match token id */
 			if (tokens[j].token != le32_to_cpu(elem->token))
 				continue;
+			else
```

Comment:
no need for an "else" - the below code is also in the same "else" case (implicitly). Your `found++;` is just one of the statements, that will be executed, if the branch isn't taken.

Reply:
ok